docs/scripts/basic.md

```diff
@@ -0,0 +1,70 @@
+
+We shall first plan together based on the complete interest of the group what we want to work on today.
+We shall then make a plan for how we want to work together today.
+Using our plan we shall practice investigating and researching different ways the issue can be approached.
+We shall make up questions that we could ask to specific people that might help us make places that we all want to live together in. 
+
+My purpose here as a facilitator is to make sure that we all get what matters to us as earthlings who want to understand each other and be represented by our elected officials. I am also here to make sure that we work towards our purpose and not against it. I may at times have to step in to remind members of the group the ground rules.
+
+## Container
+
+We are now going to make a container of agreements and expectations so that we can all work towards our objective together effectively.
+
+* Please allow everyone to speak without interruption.
+* We also ask that we not share what our personal political views are to the group. We can legitimately achieve our purpose without sharing our individual views publically although we shall have a chance to write down privately our current satisfaction. We then review group priorities together.
+* We ask that everyone treat one another respectfully: no name calling, personal attacks, misgendendering, racism, sexism, ablism, ageism, or descrimination is allowed.
+* We ask that you actively request consent rather than assume consent.
+* If we need to keep things flowing, I may step in to put things right.
+
+Are there any questions or potential additions to our container?
+
+Does everyone agree to our container? Thumbs up thumbs down?
+
+*Shows a thumbs up and waits for everyone to decide*
+
+## Assess Interest
+
+*Passes out paper if not already passed out*
+Does everyone have paper? We are now going to assess what everyone in the group wants to work on without everyone having to know everyone's personal opinions. I'm going to ask you two questions and I want you to write your answers on the paper. Are you ready?
+*Shows a "thumbs up."*
+
+On a scale of one to ten, one being this presidency not been good at all to ten being that there could not be a more perfect presidency, how satisfied are you with the current president?
+
+*Waits till everyone looks done.*
+
+Are you ready for the next question?
+
+*Shows a thumbs up?*
+
+
+
+May I collect your answers now?
+
+*The facilitator collects all the answers."
+
+In order to make it difficult 
+
+## ETC
+ 
+Everyone can participate as much or as little as you want. Does anyone know that they don't want to participate or they want to watch until you raise your hand to participate? By a show of hands does anyone already know that you want to participate? Would anyone 
```

Comment:
Everyone can participate as much or as little as you want. Does anyone know that they don't want to participate? 
Does anyone prefer to watch until you raise your hand to participate?  Does anyone already know that you want to participate?